packages/components/Switch/src/Switch.styling.ts

```diff
@@ -0,0 +1,95 @@
+import { switchName, SwitchTokens, SwitchSlotProps, SwitchProps } from './Switch.types';
+import { UseStylingOptions, Theme, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, shadowStyles } from '@fluentui-react-native/tokens';
+import { defaultSwitchTokens } from './SwitchTokens';
+
+export const switchStates: (keyof SwitchTokens)[] = [
+  'toggleOn',
+  'toggleOff',
+  'before',
+  'after',
+  'above',
+  'hovered',
+  'focused',
+  'pressed',
+  'disabled',
+];
+
+export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, SwitchTokens> = {
+  tokens: [defaultSwitchTokens, switchName],
+  states: switchStates,
+  slotProps: {
+    root: buildProps(
+      (tokens: SwitchTokens) => ({
+        style: {
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: tokens.flexDirection,
+          alignSelf: 'flex-start',
+          minHeight: 28,
+          minWidth: 40,
+          padding: 6,
+        },
+      }),
+      ['flexDirection'],
+    ),
+    toggleContainer: buildProps(
+      (tokens: SwitchTokens) => ({
+        style: {
+          display: 'flex',
+          flexDirection: tokens.toggleContainerFlexDirection,
+          alignItems: 'center',
+        },
+      }),
+      ['toggleContainerFlexDirection'],
+    ),
+    track: buildProps(
+      (tokens: SwitchTokens, theme: Theme) => ({
+        style: {
+          flexDirection: 'row',
+          height: 20,
+          borderRadius: 50,
+          width: 40,
+          backgroundColor: tokens.background,
+          borderColor: tokens.stroke,
+          borderWidth: 1,
```

Comment:
I'm not sure you need to add here borderColor/Width/Radius because you have then `...borderStyles.from(tokens, theme)`

Reply:
If there are different `borderRadius` for more than one slot I'm guessing I wouldnt be able to use `...borderStyles.from(tokens, theme)`? I think I can get borderColor and borderWidth out of the styles and instead be applied through `...borderStyles.from(tokens, theme)`. But I don't think I can do the same for `borderRadius`. Would it be better to just remove `...borderStyles.from(tokens, theme)`?

Reply:
Why won't you be able to use `...borderStyles.from(tokens, theme)`? It has the same properties you need

Reply:
Sorry I assumed I had to remove `borderRadius` on the other slot as well. But I can just leave that slot as it is and update the track slot. I've made the changes now :)